tests/scripts/run_benchmarks.sh

```diff
@@ -43,7 +43,11 @@ cat benchmark_${LASTCOMMIT}.csv
 
 echo "===> Install components for benchmark analysis ..."
 
-pip3 install matplotlib==3.7.1 numpy==1.24.2 pandas==1.5.3
+if python3 -c 'import sys; sys.exit(not (sys.version_info.major > 3 or (sys.version_info.major == 3 and sys.version_info.minor >= 9)))' ; then
```

Comment:
Right, we should set this script up again...

Reply:
What do you mean? This seemed to be running in the CI.

Reply:
This is not related to this PR, but the display of the becnhmarks has been broken for ages...

Reply:
Ah, okay :)